File: C#/백준/Bronze/33191. Yalda/Yalda.cs
using System;

namespace q{
    class w{
        static void Main(){
            int m = int.Parse(Console.ReadLine());
            int[] a = new int[3];
            for (int i=0;i<3;i++){
                a[i] = int.Parse(Console.ReadLine());
            }
            if (a[0]<=m){
                Console.WriteLine("Watermelon");
            }
            else if(a[1]<=m){
                Console.WriteLine("Pomegranates");
            }
            else if(a[2]<=m){
                Console.WriteLine("Nuts");
            }
            else{
                Console.WriteLine("Nothing");
            }
        }
    }
}

File: C++11/백준/Bronze/33191. Yalda/Yalda.cc
#include <iostream>
using namespace std;

int main(){
    int m;
    cin >> m;
    int a[3];
    for (int i=0;i<3;i++){
        cin >> a[i];
    }
    if(a[0]<=m){
        cout << "Watermelon";
    }
    else if(a[1]<=m){
        cout << "Pomegranates";
    }
    else if(a[2]<=m){
        cout << "Nuts";
    }
    else{
        cout << "Nothing";
    }
}